package.json

```diff
@@ -10,12 +10,12 @@
     "start:dev": "nodemon index.js",
     "lint": "./node_modules/.bin/eslint . --fix",
     "test:watch": "jest --watch",
-    "generator": "npx sequelize model:generate --name Social_medias --attributes username:string",
-    "migration": "npx sequelize db:migrate",
-    "create:seeds": "npx sequelize-cli seed:generate --name demo-socialMedias",
-    "dropMigration": "npx sequelize db:migrate:undo:all",
-    "undo:seed": "npx sequelize-cli db:seed:undo:all",
-    "seed": "npx sequelize-cli db:seed:all",
+    "generator": "./node_modules/.bin/sequelize  model:generate --name Mentees_chioces --attributes username:string",
```

Comment:
kindly take the node_modules out and replace it back with npx

Reply:
We will have to discuss this and agree on it.